src/main/groovy/com/xebialabs/gradle/xldeploy/DeployTask.groovy

```diff
@@ -39,13 +38,19 @@ class DeployTask extends BaseDeploymentTask {
   @SuppressWarnings("GroovyUnusedDeclaration")
   public void executeDeployment() {
     try {
+      if(xldExtension.xldDeployLogLevel) {
```

Comment:
Can't the default value be define in XldeployPluginExtension?

Reply:
Yes.  I'll move the default configuration to the extension file and add a test to make sure it is set by default to LogLevel.INFO